services/src/server.rs

```diff
@@ -204,19 +202,19 @@ mod tests {
 
     async fn queries(shutdown_tx: Sender<()>) {
         let web_config: config::Web = get_config_element().unwrap();
-        let base_url = format!("http://{}", web_config.bind_address);
+        let base_url = Url::parse(&format!("http://{}", web_config.bind_address)).unwrap();
```

Comment:
we need to enfore the trailing / here as well or not?

Reply:
c5b312d